### DISCUSSION TITLE
fpdf 1.7.2 todo fpdf2

### DISCUSSION THREAD

**Discussion by pabletesc** [py-pdf/fpdf2#1164] [Q&A]
Hello.

I am using Fpdf ( 1.7.2), and I have uninstalled it to use fpdf2. It just does not work for me, I have followed the examples and I have been able to verify that it instantiates the same, but it gives me failures.

I am using python 3.12

Thanks and greetings

**Comment by Lucas-C** [comment]
Hi @pabletesc

I'm one of `fpdf2` maintainers.
I'd be happy to try to help you, but the information you provided so far does not allow me to help you much 😅

I'm going to assume that you are using `pip` as a package manager, but please correct me if that's wrong.
Could you please provide:
* the output of the command: `pip install --upgrade fpdf2`
* the output of the command: `python -c "from fpdf import FPDF_VERSION; print(FPDF_VERSION)"`
* the full error output when you tried to execute the examples that you mentioned "failed"
* the resulting PDF if some files were produced

**Comment by pabletesc** [comment]
HolaYa he conseguido me funcione, solo tengo un problema que no consigo hacer que imprima el carácter €.Muchas gracias Enviado desde mi iPhoneEl 13 may 2024, a las 10:57, Lucas Cimon ***@***.***> escribió:﻿
Hi @pabletesc
I'm one of fpdf2 maintainers.
I'd be happy to try to help you, but the information you provided so far does not allow me to help you much 😅
I'm going to assume that you are using pip as a package manager, but please correct me if that's wrong.
Could you please provide:

the output of the command: pip install --upgrade fpdf2
the output of the command: python -c "from fpdf import FPDF_VERSION; print(FPDF_VERSION)"
the full error output when you tried to execute the examples that you mentioned "failed"
the resulting PDF if some files were produced

**Comment by Lucas-C** [comment]
I do not speak Spanish so there is an automated translation of your answer:

> I have already managed to get it to work, I just have a problem that I can't get it to print the character.

My understanding is that you do not need help regarding the installation of `fpdf2`.

If you want further help regarding using this library, please take the time to open a dedicated issue, and follow those guidelines to provide enough information for us to help you: https://stackoverflow.com/help/minimal-reproducible-example

**Comment by pabletesc** [comment]
Ok, sorry.Thanks Enviado desde mi iPhoneEl 13 may 2024, a las 17:45, Lucas Cimon ***@***.***> escribió:﻿
Closed #1164 as resolved.
